apps/ima/views/helper-declaration.html

```diff
@@ -1,7 +1,7 @@
+<title>{{#t}}pages.helper-declaration.helper{{/t}} – GOV.UK</title>
```

Comment:
I don't think this is needed here, the title is already showing as 'Declaration'

Reply:
Removed